docworks-dts/test/services/DotConventionNamespace.service.json

```diff
@@ -0,0 +1,6406 @@
+{
+  "name": "Orders",
```

Comment:
this file is huge, can u please remove some operations (6) and messages (107) 🙏

Reply:
sure